src/components/ol/layer/Tile.tsx

```diff
@@ -66,12 +66,16 @@ export function OSMBlackAndWhite(): JSX.Element {
 
 interface TileProps
   extends MapComponentProps,
-    OlTileLayerOptions<OlTileSource> {}
+  OlTileLayerOptions<OlTileSource> { }
 
 export class Tile extends MapComponent<OlTileLayer<OlTileSource>, TileProps> {
   addMapObject(map: OlMap): OlTileLayer<OlTileSource> {
     const layer = new OlTileLayer(this.props);
     layer.set("id", this.props.id);
+    const source = layer.getSource() as OlTileSource & { crossOrigin?: string };
+    if (source && 'crossOrigin' in source) {
+      source.crossOrigin = "Anonymous";
+    }
```

Comment:
Please add a comment that explains why this is needed. Add a link to the source too.